### PR TITLE
[10.0][MIG] mailbox_replica app

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
image

### CODE REVIEW

File: mailbox_replica/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Eficent/mailbox_replica>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and
+welcomed feedback `here <https://github.com/Eficent/
+mailbox_replica/issues/new?body=module:%20 mail_sent%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (MiquelRForgeFlow) [inline mailbox_replica/README.rst:33]: this is old shit

File: mailbox_replica/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        'partner_mail_message_view',
+        'fetchmail_fetch_missing',
+        'mail_create_partner',
+        'mail_all',
+        'web_polymorphic_selection',
+    ],
+    'author': 'Eficent',
+    'summary': """
+      Mailbox replica app
+""",
```
Comment (MiquelRForgeFlow) [inline mailbox_replica/__manifest__.py:23]: ugly

Comment (AaronHForgeFlow) [reply mailbox_replica/__manifest__.py:23]: Excuse me Mr Shakespeare

**Pull request merged**